2nd-gen/test-functions-mocha/functions/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "functions",
+  "scripts": {
+    "lint": "eslint --ext .js,.ts .",
+    "build": "",
+    "serve": "npm run build && firebase emulators:start --only functions",
+    "shell": "npm run build && firebase functions:shell",
+    "start": "npm run shell",
+    "test": "mocha *.test.js",
+    "deploy": "firebase deploy --only functions",
+    "logs": "firebase functions:log"
```

Comment:
please add the `compile` script here. Can copy from the Jest sample

Reply:
Updated!